modules/vaos/app/services/vaos/messages_service.rb

```diff
@@ -31,10 +31,31 @@ def get_messages(request_id)
       end
     end
 
+    def post_message(request_id, request_object_body)
+      with_monitoring do
+        params = VAOS::MessageForm.new(user, request_id, request_object_body).params
+        response = perform(:post, messages_url(request_id), params, headers)
+
+        if response.status == 200
+          {
+            data: OpenStruct.new(response.body),
```

Comment:
Any need for a full response object here?

Reply:
no need for a full response object, but I think in future refactor I'll change to that. I think the validations and stuff should happen at the controller level. It's a poor abstraction but one based on precedent in other services all over the platform where the service is responsible for normalizing the data both in and out. It's tricky, but a separation of concerns seems appropriate.

Reply:
> Can we add a test to definitively show this?

I wish I had recorded the cassette for that. Would have to go back and re-record it or mock up a fake one. It's fairly obvious as a bug when looking at the VCR cassettes though. And when VAMF makes the change we will likely have to fix as part of a v2, so i'm ok with no spec for now if you are.